.github/workflows/update-search-and-contributors.yml

```diff
@@ -0,0 +1,40 @@
+name: "Update search and contributors"
+
+on:
+  push:
+    paths:
+      - "contents/**"
+    paths-ignore:
+      - "contents/**/metadata.json"
+    branches:
+      - main
+
+jobs:
+  update:
+    name: Update
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: main
+
+      - name: Install
+        uses: ./.github/composite-actions/install
+
+      - name: Run update script
+        run: |
+          pnpm gen:search
+          pnpm gen:contributors
+
+      - name: Create PR
+        uses: peter-evans/create-pull-request@v6
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          base: main
+          delete-branch: true
+          commit-message: "ci: updated search and contributors"
+          title: "Updated search and contributors"
+          branch: changeset-release/main
```

Comment:
Change to one of the following candidate branch names.

- `chore/contributors`
- `build/contributors`
- `ci/contributors`